### PR TITLE
A new correlationId should be set for each upload to avoid conflict i…

### PR DESCRIPTION
**Pull request by dczulada**
…n patient cache

### CODE REVIEW

File: config/initializers/mongo.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module QME
+  patient_cache = Mongoid.default_client['patient_cache']
+  patient_cache.indexes.create_one({ 'value.measure_id' => 1, 'value.sub_id' => 1, 'value.effective_date' => 1, 'value.patient_id' => 1 },
```
[the remark this answers is not in the document]
Comment (rbclark) [reply config/initializers/mongo.rb]: Disregard, turns out I caused that with my own testing of this exact bug 😨

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Style/FrozenStringLiteralComment:
+  Enabled: false
```
Comment (okeefm) [inline .rubocop_todo.yml]: Is there a reason we're disabling this cop? This has been added to most files.

Comment (okeefm) [reply .rubocop_todo.yml]: @dczulada this should be good to go once this `.rubocop_todo.yml` gets deleted.

**Comment by dczulada** [comment]
@okeefm to_do is gone

**Pull request merged**